recipe/bld.bat

```diff
@@ -1,4 +1,11 @@
-move bin\* %LIBRARY_BIN%\
+REM Move only desired files as the downloaded package contains even Qt5 dlls
+move bin\cmake.exe %LIBRARY_BIN%\
+move bin\cmcldeps.exe %LIBRARY_BIN%\
+move bin\cpack.exe %LIBRARY_BIN%\
+move bin\ctest.exe %LIBRARY_BIN%\
+move bin\msvcp120.dll %LIBRARY_BIN%\
```

Comment:
Even if they are not installed without visual studio, we should have a separate package for them. This is because many packages may overwrite that same path. The CMake package should then depend on that one.